### PR TITLE
Otterspace: Using updated graph model [otterspace-badges]

### PR DESCRIPTION
**Pull request by rahulrumalla**
The upstream subgraph was updated

### CODE REVIEW

**Comment by rahulrumalla** [comment]
Please do no merge this just yet. If it can just be reviewed for now, that'd be great.

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
Let me know when to merge :)

**Comment by rahulrumalla** [comment]
Hi @ChaituVR this is ready to re-review and merging now..

File: src/strategies/otterspace-badges/index.ts [hunk 1]
```diff
@@ -52,7 +52,8 @@ function getBadgeWeight(specs: any[], badgeSpecID: string): number {
   if (specs && specs.length > 0) {
     const specConfig = specs.find((spec: any) => spec.id === badgeSpecID);
     badgeWeight =
-      specConfig && !isBadgeExpired(specConfig.expiresAt)
+      specConfig &&
+      isBadgeActive(specConfig.status, specConfig.metadata?.expiresAt || null)
```
Comment (ChaituVR) [inline src/strategies/otterspace-badges/index.ts:56]: @rahulrumalla There are  open proposals, 


```JSON
      {
        "id": "0x034ef9bb0c706c9a0c01afe6d98864ac94e5c037efb0e0027183de907d1a5a37",
        "space": {
          "id": "emilyfurlong.eth"
        },
        "state": "active"
      },
      {
        "id": "0x81eff3b728a65529290bf55b0f9d5293da218bc4f7891baa38c9079975ed2003",
        "space": {
          "id": "otterspace.mirror.xyz"
        },
        "state": "active"
      },
```
But they are not using `expiresAt` so I think it should be fine.?

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**